docker.go

```diff
@@ -0,0 +1,72 @@
+package llblib
+
+import (
+	"context"
+
+	"github.com/moby/buildkit/client/llb"
+	"github.com/moby/buildkit/frontend/dockerfile/dockerfile2llb"
+	"github.com/moby/buildkit/solver/pb"
+	specsv1 "github.com/opencontainers/image-spec/specs-go/v1"
+)
+
+// DockerfileOpts alias dockerfile2llb.ConvertOpt
+type DockerfileOpts = dockerfile2llb.ConvertOpt
+
+// DockerfileOption can be used to modify a Dockerfile request.
+type DockerfileOption interface {
```

Comment:
Any particular reason to use this pattern instead of straight `func(*DockerfileOpts)`? Is it for compatibility with dockerfile2llb?

Reply:
just trying to be consistent with all the options used in `llb`.  It does allow for a single option implementing multiple setters (like the NullOption) which reminds me, I need to add this to NullOption...